### PR TITLE
Fix asan

### PR DESCRIPTION
**Pull request by Vane11ope**

### CODE REVIEW

**Comment by Vane11ope** [comment]
sys/asan and V! then entropy/entropy Fire, keep pressing t would crash r2 and it's gonna be fixed with this.

File: libr/core/cmd_print.c [hunk 1]
```diff
@@ -3489,7 +3489,7 @@ static void cmd_print_bars(RCore *core, const char *input) {
 					break;
 				case 'z':
 					if ((IS_PRINTABLE (p[j]))) {
-						if (p[j + 1] == 0) {
+						if (!(&p[j + 1])) {
```
Comment (radare) [inline libr/core/cmd_print.c]: wtf this cant be null how to repro this because the fix looks wrong to me

Comment (Vane11ope) [reply libr/core/cmd_print.c]: you gave me this error and i wrote how to repro in description.

**Comment by radare** [comment]
i pushed the right fix. this is wrong

**Pull request closed without merging**